components/src/StatusTag/index.tsx

```diff
@@ -0,0 +1,17 @@
+
+import React, { ReactNode } from 'react'
+import { Badge, Flex } from 'theme-ui'
+
+
+const StatusTag = ({children, variant}: Prop) => {
+    return (
+    <Badge variant= {`statusTag.${variant}`}><Flex sx= {{alignItems: 'center' , height: '100%'}}>{children}</Flex></Badge>
+)}
```

Comment:
Since you return the value straight in the arrow function, you can remove the curly brackets (`{}`) and the `return` keyword (basically change from `=> { return ( ... ) }` to `=> (...)` :rocket:

Reply:
Could you also make it so that the `variant` is the default value used as `children` in the component? That way, if someone would just create `<StatusTag variant="stable" />`, for instance, it'd use "stable" as the value displayed in the badge as well. I think simply using `children ?? variant` inside the component should work :pray: